src/components/ContactsList/ContactRow.jsx

```diff
@@ -23,13 +28,19 @@ const ContactName = ({ firstname, lastname }) => (
 );
 ContactName.propTypes = {
   firstname: PropTypes.string,
-  lastname: PropTypes.string
+  lastname: PropTypes.string,
+  me: PropTypes.bool
 };
 ContactName.defaultProps = {
   firstname: "",
-  lastname: ""
+  lastname: "",
+  me: false
 };
 
+const MeMarker = (props, { t }) => (
```

Comment:
Not really fan of this name, but it is hard to find a good name, maybe a `myselfTag` or `myselfLabel`.